README.md

```diff
@@ -2,6 +2,8 @@
 
 This Visual Studio Code extension provides experimental, advanced, and optional features to extend the Jupyter notebook experience in VS Code.
 
+This extension is intended to supplement the features provided by the main [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter). As this extension is providing experimental features there is no set roadmap for how features will be added or removed from it. Features may be added or removed at any time and there is no guarantee of future support by virtue of inclusion in the extension now. Features in this extension that see high usage may be considered for removal from this extension and inclusion in the main Jupyter VS Code extension. In that case, best efforts will be made to notify of the change and migrate settings if possible.
```

Comment:
This is the wording that I'm most concerned about (Kai expressed similar concern). To me, the open source licensing here makes it feel lower risk. If we have a feature in here that we don't want to support, there is no reason that a customer can't pick it up and release it as a separate extension.

Reply:
Looping in @greazer and @minsa110 specifically here for wording.

Reply:
@rebornix as well.

Reply:
missing comma: 

> As this extension is providing experimental features**,**